### PR TITLE
Fixes #20489: Add inventory plugin

### PR DESCRIPTION
**Pull request by julienbriault**
https://issues.rudder.io/issues/20489

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -234,6 +234,25 @@ the roles listed above.
...
             repository_password: "my_password"
 ```
+## Plugin usage
+
+### Inventory plugin
+
+The inventory plugin allows you to retrieve your inventory (list of Rudder nodes) in the Ansible inventory.
+
+#### Plugin default parameters
+- `cache`: Enable caching (default: `False`)
```
Comment (Fdall) [inline README.md:244]: I did not see where those were configurable in the code, are they? You often do not want to run the inventory from the Rudder server but will instead run it from your personal workstation.
The api uri also needs to be configurable.

Comment (julienbriault) [reply README.md:244]: I just corrected it so that the values are properly configurable.

Comment (Fdall) [reply README.md:244]: :+1: Looks good, just add an example in the README with a proper inventory call with override of the uri for instance.

Comment (julienbriault) [reply README.md:244]: It is now done !

**Review by Fdall — APPROVED** [review]

**Comment by amousset** [comment]
Tests are not passing

**Pull request closed without merging**